views/doctorform.ejs

```diff
@@ -40,7 +53,19 @@
     <div class="row form-group input_group">
 
       <div class="col-sm-12">
-       <input id="specialization" type="specialization" name="specialization" class="form-control" placeholder="specialization">
+       <!-- change input type to select> -->
+      <select  class="form-control" id="specialization" name="specialization">
```

Comment:
Instead of manually writing out all of these options, you can have a list of options in your component and then just render them here. This will save you from having to write ```<option> . . . </option>``` each time, and will allow you to have a single place where these options are defined.

As an example, imagine if you had a second select element in your application that used these options. If you needed to add an option to the list, you would need to add the option in two places. If, instead of listing the options out in the html, you had them defined in your component file, you would only need to add the option in one place.

This is an example of the old rule to keep your code "DRY" (stands for "Don't Repeat Yourself"). If you avoid repeating yourself, you write less code, and so it is easier to edit it and fix mistakes in it.